File: 1.x/src/com/relevantcodes/extentreports/LogSettings.java
/*
* Copyright (c) 2015, Anshoo Arora (Relevant Codes).  All rights reserved.
* 
* Copyrights licensed under the New BSD License.
* 
* See the accompanying LICENSE file for terms.
*/


package com.relevantcodes.extentreports;

public abstract class LogSettings {
	protected static Boolean displayCallerClass = true;
	protected static String logTimeFormat = "hh:mm:ss";
	protected static String logDateFormat = "mm-dd-yyyy";
	protected static String logDateTimeFormat = logDateFormat + " " + logTimeFormat;
}

File: dotnet/ExtentReports/LogSettings.cs
// ***********************************************************************
// Copyright (c) 2015, Anshoo Arora (Relevant Codes). All rights reserved.
//
// Copyrights licensed under the New BSD License.
//
// See the accompanying LICENSE file for terms.
// ***********************************************************************

namespace RelevantCodes.ExtentReports
{
    internal abstract class LogSettings
    {
        protected static string LogTimeFormat = "HH:mm:ss";
        protected static string LogDateFormat = "yyyy-MM-dd";
        protected static string LogDateTimeFormat = LogDateFormat + " " + LogTimeFormat;
    }
}
